live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -351,13 +351,18 @@
   register: result
   until: result is succeeded
 
-#
-# Increase the amount of memory reserved for the crash kernel.
-# Empirically, it seems we need about 512M to get it to boot. If the
-# system has less than 1G, disable crash dumps rather than reserving
-# half of memory for the crash kernel.
-#
 - lineinfile:
-    path: /etc/default/grub.d/kdump-tools.cfg
+    path: "/etc/default/grub.d/{{ item.file }}.cfg"
```

Comment:
What are the tradeoffs between making the change here, vs in `/etc/modprobe.d/zfs.conf`?

Reply:
I thought that that didn't work with zfs on root because zfs is loaded before the root filesystem was mounted. However, I just did a bit more research, and it looks like I just needed to regenerate the initramfs for the change to take effect. I'll update this to use `/etc/modprobe.d`.